### ISSUE TITLE
Als stel wil ik duidelijke teksten bij optie van trouwen

### ISSUE THREAD

**Issue by JanWillemKooi** [VNG-Realisatie/huwelijksplanner#299]
zodat ik weet wat ik kan kiezen.

- [x] Zorg dat de beschrijving duidelijk voor iedereen in het team is
- [x] Koppel de juiste labels
- [ ] Koppel eventueel aan een epic (door # en het nummer toe te voegen)
- [x] Voeg alle taken die nodig zijn voor de user story toe.

- 3 teksten aanleveren voor 'Eenvoudig', 'Gratis', en 'Uitgebreid'  (inhoudelijke mensen)
- teksten in de applicatie opnemen (techniek)
- testen of teksten goed zijn ingevoerd en duidelijk zijn (project medewerkers)

**Comment by CMasselink** [comment]
Technisch kan dit. Gaat nu vooral om inhoud van de tekst.
@JeroenduC Kan jij hier naar kijken?

**Comment by JeroenduC** [comment]
Ja, maar het lijkt mij niet zo handig dat ik de tekstschrijver wordt in deze. Misschien slimmer om hier te kijken naar de teksten die al bestaan bij de gemeentes en die te gebruiken?
